staking/slash/double-sign.go

```diff
@@ -473,9 +474,7 @@ func Apply(
 			RawJSON("slash", []byte(slash.String())).
 			Msg("about to update staking info for a validator after a slash")
 
-		if err := state.UpdateValidatorWrapper(
-			snapshot.Validator.Address, current,
-		); err != nil {
+		if err := current.SanityCheck(staking.DoNotEnforceMaxBLS); err != nil {
```

Comment:
I think you turned off the check so it was intentional , then can remove the bls const hack

Reply:
what do you mean turn of the bls const hack? You mean the SanityCheck is not needed?

Reply:
oh wait, I was wrong, I saw wrng on earlier comment by ganesh, I though this part of logic in the check itself was removed